### DISCUSSION TITLE
error with Introduction to scRNA-seq integration vignettes

### DISCUSSION THREAD

**Discussion by Ibisanmi1** [satijalab/seurat#7384] [Q&A]
please i am having error running Introduction to scRNA-seq integration vignette. unable to download the data ifnb for the anaylysis

```{r, include = FALSE}
options(SeuratData.repo.use = "http://satijalab04.nygenome.org")
```

```{r data}
library(Seurat)
library(SeuratData)
library(patchwork)
```

```{r installdata, eval=FALSE}
# install dataset
InstallData('ifnb')
```
Error: No access to remote SeuratData repository, unable to install new datasets

**Comment by jack7481** [comment]
Hope this help you right.
1. Download the ifnb.SeuratData as a R package from source.
http://seurat.nygenome.org/src/contrib/ifnb.SeuratData_3.1.0.tar.gz
2. Access in RSstudio in your own drive
install.packages("**download.dir/ifnb.SeuratData_3.1.0.tar.gz**", repos = NULL, type = "source")
3. Access ifnb.SeuratData as a R package
library(ifnb.SeuratData)
4. Load it!
LoadData("ifnb")

Good luck!
